src/calendar/styles.scss

```diff
@@ -18,11 +18,12 @@
 .calendar {
   display: block;
   // IE11 does not calculate the height correctly when in nested flex containers (@see https://github.com/philipwalton/flexbugs#flexbug-3)
-  width: awsui.$size-calendar-grid-width;
+  inline-size: awsui.$size-calendar-grid-width;
```

Comment:
IE11 and `inline-size` make no sense in one context. There is probably an opportunity to remove some unneeded styles or update the comment if this style needed for other browsers too